exercises/practice/matching-brackets/.meta/config.json

```diff
@@ -1,6 +1,14 @@
 {
   "blurb": "Make sure the brackets and braces all match.",
-  "authors": [],
+  "authors": [
+    "johnngugi"
+  ],
+  "contributors": [
+    "amscotti",
+    "devkabiir",
+    "ErikSchierboom",
+    "Stargator"
+  ],
```

Comment:
```suggestion
  "contributors": [
  ],
```

Reply:
Do we list users as contributors for renaming? `bracket-push` was renamed to `matching-brackets` with this PR, https://github.com/exercism/dart/pull/188

Reply:
cc @ErikSchierboom

Reply:
Based on a conversation, the OP of the PR says (my emphasis):
> Contributors should be people who have **substantially contributed** to that exercise. PRs for typos or multiple-exercise PRs do not automatically mean someone should be considered a contributor to that exercise. Those non-exercise-specific contributions will get recognition through Exercism's reputation system in v3.

After looking at the commit in question, I would think of that as significant. While test cases or the example doesn't change, it was a significant effort in my opinion.

Reply:
```suggestion
  "contributors": [
    "amscotti"
  ],
```